faqs/galaxy/collections_rename.md

```diff
@@ -4,7 +4,6 @@ area: collections
 box_type: tip
 layout: faq
 contributors: [delphine-l, martenson, shiltemann, hexylena]
-
```

Comment:
huh, this empty line was causing the bad rendering? nice catch! We should probably add a check for that to the github testing framework.

Reply:
This should be tested though if it actually makes a difference and removes the unnecessary info. I didn't see anything wrong with it besides this inconsistency compared to the other files.

Reply:
I just built the website locally and it is indeed fixed, so nice find :+1: